helm/video/values.yaml

```diff
@@ -98,11 +98,17 @@ background_services:
   # - It consumes messages from the message broker (the video status, send by the encoder)
   # - Updates the video status in the upload database
   - name: "upload-message-broker-consumer"
-    type: rabbitmq-consumer
```

Comment:
why change the scenario?

Reply:
After discussion, I added back rabbitmq aspect